# Segmentation Info
Header header

int32[] ring_start_idx
int32[] ring_end_idx

uint32[] point_label
uint32[] point_column
float32[] point_depth

sensor_msgs/PointCloud2 extracted_cloud
